public/playersBetController.js

```diff
@@ -0,0 +1,140 @@
+        <button type='button' class= 'bet-controls__btn btn--50'>
+        50
+        </button>
+        <button type='button' class= 'bet-controls__btn btn--100'>
+        100
+        </button>    
+        <button type='button' disabled class= 'bet-controls__btn btn--clear'>
+        Clear
+        </button>
+        <button type='button' disabled class= 'bet-controls__btn btn--bet'>
+        Confirm bet
+        </button>
+    </div>    
+      `;
+  }
+
+  rederBetControls() {
+    this.controlsContainerEle.insertAdjacentHTML(
+      "beforeend",
+      this.controlsHtml()
+    );
+  }
+
+  initBetController() {
+    this.initPlayers();
+    this.rederBetControls();
+    this.selectDomElements();
+    this.addEventListeners();
+  }
+
+  initPlayers() {
+    this.players = this.board.players;
+    this.updateActivePlayer();
+  }
+
+  selectDomElements() {
+    this.betControls = document.querySelector(".bet-controls");
+    this.bet25 = this.controlsContainerEle.querySelector(".btn--25");
+    this.bet50 = this.controlsContainerEle.querySelector(".btn--50");
+    this.bet100 = this.controlsContainerEle.querySelector(".btn--100");
+    this.clear = this.controlsContainerEle.querySelector(".btn--clear");
+    this.currentPotCont =
+      this.controlsContainerEle.querySelector(".bet-controls__pot");
+    this.curretBetCont =
+      this.controlsContainerEle.querySelector(".bet-controls__bet");
+    this.confirm = this.controlsContainerEle.querySelector(".btn--bet");
+  }
+
+  addEventListeners() {
+    this.bet25.addEventListener("click", this.betting.bind(this));
+    this.bet50.addEventListener("click", this.betting.bind(this));
+    this.bet100.addEventListener("click", this.betting.bind(this));
+    this.clear.addEventListener("click", this.clearBet.bind(this));
+    this.confirm.addEventListener("click", this.confirmBet.bind(this));
+  }
+
+  updateActivePlayer() {
+    this.activePlayer = this.players[this.currentPlayerTurn];
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.add("active");
+  }
+
+  isLastPlayerTurn() {
+    const lastPlayerPosition = this.players.length - 2;
+    return lastPlayerPosition === this.currentPlayerTurn;
+  }
+
+  nextPlayer() {
+    document
+      .querySelector(`#${this.activePlayer.name} .card`)
+      .classList.remove("active");
+    if (this.isLastPlayerTurn()) {
+      this.betControls.remove();
+      this.board.sartWithGameDealing();
+    } else {
+      this.currentPlayerTurn++;
+      this.updateActivePlayer();
+      this.currentPotCont.innerHTML = `Pot: ${this.activePlayer.pot}`;
+      this.curretBetCont.innerHTML = `Current bet: ${this.currentBet}`;
+    }
+  }
+
+  betting(e) {
+    this.clear.disabled = false;
+    this.confirm.disabled = false;
+    const betAddition = e.target.innerText;
+    if (this.activePlayer.pot - betAddition < 0) {
+      return;
+    }
+    this.activePlayer.pot -= betAddition;
+    this.currentBet += 1 * betAddition;
```

Comment:
To convert a string to a number it is best to do:
```suggestion
    this.currentBet += Number(betAddition)
```